DXM-180 Updated the instruction and the screenshots with new UI. (#3)
qualtrics/qualtrics-digital-ios-sdk: README.md

The release notes for each version can be found here: [Release Notes](https://api.qualtrics.com/sdks/ZG9jOjgwNTgzNjc-i-os-mobile-app-sdk-releases)

## Installing the package through Swift Package Manager

> Add a dependency on qualtrics-digital-ios-sdk to your ios application through Xcode by following these steps:

Reviewer: ```suggestion
Add a dependency on qualtrics-digital-ios-sdk to your iOS application through Xcode with the following steps:
```

#### 1. Navigate to `File`->`Add Packages...`
![step 1](https://github.com/qualtrics/qualtrics-digital-ios-sdk/blob/main/ReadMeFiles/step1.png?raw=true)

#### 2. Search for the package repository url: `https://github.com/qualtrics/qualtrics-digital-ios-sdk` in the top right corner:
![step 2](https://github.com/qualtrics/qualtrics-digital-ios-sdk/blob/main/ReadMeFiles/step2.png?raw=true)

#### 3. Double-click on the SDK
![step 3](https://github.com/qualtrics/qualtrics-digital-ios-sdk/blob/main/ReadMeFiles/step3.png?raw=true)

#### 4. Check the box, and click “Add Package”
![step 4](https://github.com/qualtrics/qualtrics-digital-ios-sdk/blob/main/ReadMeFiles/step4.png?raw=true)

> #### 5. Make sure the package is added (for example for application `QualtricsSDKSwiftUI` it should appear on the left menu in files structures, and look as follows):

Reviewer: ```suggestion
#### 5. Verify that the package has been added to your Project Navigator in Xcode, under `Package Dependencies` (In this example, it is directly under our example app `QualtricsSDKSwiftUI`):
```
